packages/myst-cli/src/process/site.ts

```diff
@@ -179,7 +179,7 @@ export async function writeFile(
   const toc = tic();
   const selectedFile = selectFile(session, file);
   if (!selectedFile) return;
-  const { frontmatter, ...mdastPost } = selectedFile;
+  const { frontmatter, mdast, kind, sha256, slug, references, dependencies } = selectedFile;
```

Comment:
This is just removing `file` from the written page config. I think this is ok? Previously this was writing local file paths... not useful at all for the site. (The original file is still available as a download in the export lists.)

Reply:
My thought on `file` is that it will be helpful when we sync up with an 'Edit on GitHub' which will require a local file path.

Reply:
ok so sounds like this is one mini change to revert!

Reply:
In talking through with @fwkoch I think we should actually remove `file`, it is non-relative at the moment. We can bring this back soon with a feature of `edit on github`!